config/filter.d/apache-common.conf

```diff
@@ -5,10 +5,16 @@
 
 # Load customizations if any available
 after = apache-common.local
+# Uncomment the following line if you use ErrorLog = syslog
+before = common.conf
```

Comment:
My mistake for the 1st point, sorry...
For the second, totally agree, I then just committed a fix.
Thank you 👍